support/templates/behandeling-html.hbs

```diff
@@ -77,7 +69,7 @@
         <p>
           Met als gevolg,
         </p>
-        <p property="besluit:gevolg">{{this.result}}</p>
+        <p datatype="rdf:langString" property="besluit:gevolg">{{this.result}}</p>
```

Comment:
this is not the correct solution, for it to be a valid langstring we should set the language of the the value. the datatype is then automatically a langstring. See https://www.w3.org/TR/html-rdfa/#h-specifying-the-language-for-a-literal

Reply:
Yes, you are right I'm missing the language, I was going to add it but I didn't, are you sure we don't need the datatype for marawa to pick it up?

Reply:
hopefully marawa does the right thing

Reply:
I've added a PR on marawa so it does the right thing, this will be a good testcase for it :)